libsolutil/CDCL.cpp

```diff
@@ -230,9 +254,56 @@ std::pair<Clause, size_t> CDCL::analyze(Clause _conflictClause)
 	return {move(learntClause), backtrackLevel};
 }
 
-void CDCL::addClause(Clause _clause)
+void CDCL::addClause(const vector<Literal>& _lits)
 {
-	m_clauses.push_back(make_unique<Clause>(move(_clause)));
+	if (!ok) {
+		return;
+	}
+
+	Clause _clause {_lits, clauseID++};
```

Comment:
```suggestion
	Clause clause{_lits, clauseID++};
```
(underscore only for parameters) and no space in front of expression-level parentheses or similar

Reply:
Thanks! This has now been fixed :)